api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-18.xml

```diff
@@ -207,10 +207,11 @@
   <concept concept_id="162583" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2014-09-18 15:14:53.0" version="" changed_by="1" date_changed="2016-01-27 01:41:48.0"  uuid="162583AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="162798" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2014-12-19 21:45:57.0" version="" changed_by="1" date_changed="2015-01-16 19:56:53.0"  uuid="162798AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="162824" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-12 18:19:41.0" version="" changed_by="1" date_changed="2015-01-12 18:19:50.0"  uuid="162824AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="165171" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2018-07-31 10:13:02.0" version="" changed_by="1" date_changed="2018-07-31 10:13:14.0"  uuid="165171AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept_description concept_description_id="107" concept_id="108" description="A contraction of smooth muscle in the walls of the bronchi and bronchioles, causing narrowing of the lumen." locale="en" creator="1" date_created="2004-01-01 00:00:00.0" changed_by="1" date_changed="2015-11-05 01:34:22.0" uuid="107FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
   <concept_description concept_description_id="120" concept_id="121" description="Muscular pain or tenderness, especially when diffuse and nonspecific." locale="en" creator="1" date_created="2004-01-01 00:00:00.0" changed_by="1" date_changed="2015-11-05 01:37:39.0" uuid="120FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
   <concept_description concept_description_id="512" concept_id="512" description="DEF: Lesion of the skin. DESC: The history of the particular skin lesions is also important, including time and site of initial appearance, spread, change in appearance, and triggering factors. Visual inspection is the central evaluation tool; many skin diseases are diagnosed by the characteristic appearance or morphology of the lesions. REF: Merck Manual" locale="en" creator="1" date_created="2004-01-01 00:00:00.0" changed_by="1" date_changed="2016-04-27 14:18:52.0" uuid="512FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
-  <concept_description concept_description_id="927" concept_id="924" description="Antimalarial prophylactic and therapy.  Also being used for PCP treatment." locale="en" creator="1" date_created="2004-07-23 00:00:00.0" changed_by="1" uuid="927FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
```

Comment:
What actually changed here to necessitate a date_changed change?

Reply:
@dkayiwa I guess maybe the fact that changed_by is 1 and that this line didn't have a date_changed column before makes the addition of the column justified?

Reply:
changed_by without any actual change?